build/docker-hub-release.yml

```diff
@@ -0,0 +1,75 @@
+name: $(Image.Version)
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: arcus-azure/azure-devops-templates
+      endpoint: arcus-azure
+
+variables:
+  - name: 'Image.Name'
+    value: 'arcusazure/arcus-event-grid-proxy-ignore'
+  # variable 'Image.Version' is set as queue-time variable in azure devops.
+  # variable 'Image.MajorMinorVersion' is set during the first powershell step 'Determine'{major}.{minor} version'
+
+stages:
+  - stage: 'Docker'
+    jobs:
+      - job: 'Docker'
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - powershell: |
+              Write-Host "Determining '{major}.{minor}' for '$(Image.Version)'"
+
+              $index = "$(Image.Version)".IndexOf(".", 2);
+
+              $majorMinorVersion = "$(Image.Version)".Substring(0, $index);
+
+              Write-Host "Found version '$majorMinorVersion'"
+              Write-Output ("##vso[task.setvariable variable=Image.MajorMinorVersion;]$majorMinorVersion")
+            displayName: "Determine '{major}.{minor}' version"
+          - task: PowerShell@2
+            inputs:
+              targetType: 'inline'
+              script: 'docker build . --file Arcus.EventGrid.Proxy.Api/Dockerfile --tag $(Image.Name):$(Build.BuildId) --tag $(Image.Name):$(Image.MajorMinorVersion) --tag $(Image.Name):latest'
+              workingDirectory: 'src'
+            displayName: 'Build Docker image'
+          - ${{ if not(contains(variables['Build.BuildNumber'], '-')) }}:
```

Comment:
This will never be used in PRs so don't think we need this, no? 🤔

Reply:
Isn't this the condition to check for the version number; I didn't know this was related to PR?

Reply:
This is a release pipeline so this will never be triggered via a PR.

Hence why you've added triggers `none` as well.

Reply:
I removed the inline `if` line.